### PR TITLE
Fix various RPC issues

### PR DESCRIPTION
**Pull request by kentonv**
See commit descriptions.

First issue caused spurious Sentry errors.

Other two issues @geelen complained about.

### CODE REVIEW

File: src/workerd/jsg/jsvalue.h [hunk 1]
```diff
@@ -343,6 +343,7 @@ class JsObject final: public JsBase<v8::Object, JsObject> {
   void set(Lock& js, const JsValue& name, const JsValue& value);
   void set(Lock& js, kj::StringPtr name, const JsValue& value);
   void setReadOnly(Lock& js, kj::StringPtr name, const JsValue& value);
+  void setNonEnuremable(Lock& js, const JsSymbol& name, const JsValue& value);
```
Comment (jasnell) [inline src/workerd/jsg/jsvalue.h:346]: setNonEnumerable

**Review by jasnell — APPROVED** [review]
Typo in the function name but otherwise lgtm.

File: src/workerd/jsg/jsvalue.h (hunk 1 above)
Comment (kentonv) [reply src/workerd/jsg/jsvalue.h]: Funny how IDE auto-complete means these typos can spread unnoticed.

**Review by geelen — APPROVED** [review]
Fantastic! Pleased these didn't require bigger rework. Will make testing RPC APIs way cleaner 🙏

**Review by anonrig — APPROVED** [review]

**Pull request merged**